airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -138,8 +138,8 @@ export const createConnectionValidationSchema = ({
       namespaceDefinition: yup
         .string()
         .oneOf([
-          NamespaceDefinitionType.source,
           NamespaceDefinitionType.destination,
+          NamespaceDefinitionType.source,
           NamespaceDefinitionType.customformat,
```

Comment:
Same here, not strictly necessary to change the order for the validation schema, but doesn't hurt.